cdap-docs/integrations/source/mapr/index.rst

```diff
@@ -0,0 +1,51 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _mapr-configuring:
+
+=======================================
+Configuring and Installing CDAP on MapR
+=======================================
+
+.. highlight:: console
+
+This section will assist you with installing CDAP on `MapR <https://www.mapr.com>`__. CDAP
+can be installed on a MapR cluster by following the :ref:`manual instructions for installing
+CDAP <install>`. These notes are an addition to those steps.
+
+Note that as described in the :ref:`Software Prerequisites <install-software-requirements>`, 
+a configured Hadoop and HBase (plus an optional Hive client) needs to be configured
+on the node(s) where CDAP will run.
+
+- To configure a MapR Hadoop client, see the MapR documentation on `Setting Up the Client
+  <http://doc.mapr.com/display/MapR/Setting+Up+the+Client>`__.
+
+- To configure a MapR HBase client, see the MapR documentation on `Installing HBase on a Client
+  <http://doc.mapr.com/display/MapR/Installing+HBase#InstallingHBase-HBaseonaClientInstallingHBaseonaClient>`__.
+
+- To configure a MapR Hive client, see the MapR documentation on `Installing Hive
+  <http://doc.mapr.com/display/MapR/Installing+Hive>`__.
+
+A typical client node should have the ``mapr-client``, ``mapr-hbase``, and ``mapr-hive`` packages installed, and can
+be configured using the MapR `configure.sh <http://doc.mapr.com/display/MapR41/configure.sh>`__ utility.
```

Comment:
use this link: http://doc.mapr.com/display/MapR/configure.sh.  i'm assuming its the latest instead of 4.1